src/stories/Library/review/Review.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+export type ReviewProps = {
+  numberOfReviews: number;
+  metaHeadline: string;
+  hearts: 1 | 2 | 3 | 4 | 5 | 6;
+  headlineText: string;
+  bodyText?: string;
+  linkText: string;
+  linkLink?: string;
+};
+
+export const Review: React.FC<ReviewProps> = ({
+  numberOfReviews,
```

Comment:
This can be a bit confusing. We have a component called Review (singular) which can render many reviews (plural). For the sake of simplicity and the design where it is important that we manage multiple reviews properly I get why you are doing this instead of creating two components (e.g. Review and ReviewList).

Another approach might be to consider how it is handled with checkboxes: https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/release/3/src/stories/Library/Forms/checkbox/Checkbox.stories.tsx.

Reply:
Thanks for inspiration :) I adjusted it similarly.